runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPanel.java

```diff
@@ -111,36 +90,74 @@ private void populateConfig()
 				viewGroupItemsButton.addActionListener(ae -> openGroupConfigPanel(cd, configManager));
 				groupPanel.add(viewGroupItemsButton);
 				children.put(cd.getGroup().name(), groupPanel);
-				add(groupPanel);
 			});
 
```

Comment:
The body of this is moved to onSearchBarChanged because it needs to be called elsewhere. The keyListener becomes a DocumentListener because the keyLisetener is sometimes called before the TextField is actually modified, making it lag 1 character behind.